frontend_app_gallery/test/container/Gallery.spec.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { expect } from 'chai'
+import { shallow } from 'enzyme'
+import { Gallery } from '../../src/container/Gallery.jsx'
+import { APP_FEATURE_MODE } from 'tracim_frontend_lib'
+import pictures from '../fixture/content/pictures.js'
+import { debug } from '../../src/debug.js'
+import { defaultDebug } from 'tracim_frontend_lib'
+
+describe('<Gallery />', () => {
+  const folderId = 1
+  const folderName = 'folderTest'
+
+  const props = {
+    i18n: {},
+    registerCustomEventHandlerList: () => { },
+    registerLiveMessageHandlerList: () => { },
+    setApiUrl: () => { },
+    data: {
+      config: {
+        apiUrl: 'http://localhost:1337/api',
+        slug: 'gallery',
+        faIcon: 'paperclip',
+        hexcolor: '#ffa500',
+        label: 'Gallery',
+        appConfig: {
+          forceShowSidebar: false,
+          workspaceId: 0
+        },
+        history: {
+          location: {
+            search: `?folder_ids=${folderId}`
+          }
+        },
+        translation :''
+      },
+      loggedUser: {
+        ...defaultDebug.loggedUser
+      }
+    },
+    t: key => key
+  }
+
+  const stateMock = {
+    content: {
+      content_type: 'file',
+      workspace_id: 0,
+      workspaceLabel: 'workspaceTest'
+    },
+    displayedPictureIndex: 1,
+    imagesPreviews: pictures.map(picture => ({
+      src: picture.filename,
+      contentId: picture.content_id,
+      fileName: picture.filename,
+      lightBoxUrlList: [picture.filename],
+      previewUrlForThumbnail: [picture.filename],
+      rotationAngle: 0,
+      rawFileUrl: [picture.filename]
+    }))
+  }
+
+  const wrapper = shallow(<Gallery {...props} />)
+
+  describe('TLM handlers', () => {
+    describe('handleContentDeleted', () => {
```

Comment:
Why do you only have tests for `handleContentDeleted` and not `handleContentCreatedOrModifiedOrUndeleted`?

Reply:
Because handleContentCreatedOrModifiedOrUndeleted reuses the logic already in Gallery. We should discuss this.

Reply:
If the logic itself is already tested, you can at least test "if I receive a tlm, the loadGalleryList function is called".

Reply:
Right, I made more extensive tests.